hacs.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "LocalTuya",
   "domains": ["media_player", "sensor"],

```

Comment:
Should be binary_sensor, cover, fan, light, sensor and switch.

Reply:
Mind using the request changes button to make it easy for me to change it?

Reply:
Done (had totally missed that this feature actually exists in mobile UI)